doc/.gitignore

```diff
@@ -1 +0,0 @@
-Doxyfile
```

Comment:
How do we generate `Doxyfile` with `cmake`? It only needs to replace the version:

```diff
--- doc/Doxyfile.in     2022-12-12 11:24:16.156991000 +0100
+++ doc/Doxyfile        2024-04-24 14:24:56.529418000 +0200
@@ -38,7 +38,7 @@
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = @PACKAGE_VERSION@
+PROJECT_NUMBER         = 27.99.0
 
 # Using the PROJECT_BRIEF tag one can provide an optional one line description
```

Reply:
@vasild 
> How do we generate `Doxyfile` with `cmake`?

https://github.com/hebasto/bitcoin/pull/165